app/models/student.rb

```diff
@@ -1,6 +1,9 @@
 class Student < ApplicationRecord
+  
   validates :class_rank, presence: true
+  validates :has_participated, presence: true
   validates :room_draw_number, presence: true
```

Comment:
Note for later: we need to figure out how to represent nontypical roomdraw numbers, e.g. "worst" ,"last", etc. Don't know what kind of validation will be necessary at that point.